lib/src/libparodus.c

```diff
@@ -97,6 +102,38 @@ static void libpd_log ( int level, int os_errno, const char *msg, ...)
 }
 #endif
 
+//Call free in failure case for parodus_url and client_url
+static void getParodusUrl()
+{
+	const char *parodusIp = NULL;
+	const char *clientIp = NULL;
+	const char * envParodus = getenv ("PARODUS_URL");
+	const char * envClient = getenv ("CLIENT_URL");
+	char * protocol = "tcp://";
+  if( envParodus != NULL)
+  {
+    parodusIp = envParodus;
+  }
+  else
+  {
+    parodusIp = PARODUS_URL;
+  }
+  snprintf(parodus_url,sizeof(parodus_url),"%s%s",protocol, parodusIp);
+  printf("formatted parodus Url %s\n",parodus_url);
+  
+  if( envClient != NULL)
+  {
+    clientIp = envClient;
+  }
+  else
+  {
+    clientIp = CLIENT_URL;
```

Comment:
Are we allowing a default when the environment variable is not specified, or should that just be an error.